docs/node.md

```diff
@@ -69,7 +69,7 @@ Kubernetes from the node.
 ## Node Management
 
 Unlike [Pods](pods.md) and [Services](services.md), a Node is not inherently
-created by Kubernetes: it is either created from cloud providers like GCE,
+created by Kubernetes: it is either created from cloud providers like Google Container Engine,
```

Comment:
GCE = Google Compute Engine, unless you wanted to change this here?

... the more of these I read, the less I like our product naming choice